### ISSUE TITLE
Modifying assets via replace references bypasses checkout

### ISSUE THREAD

**Issue by adamfulop** [ProjectBorealis/UE4GitPlugin#150]
Hey guys, first of all, congrats for the great plugin!

One thing I've noticed it's possible to bypass the checkout limitation in the editor by using replace references when deleting assets, having modified assets without checking them out in the end.

Reproduction steps
1. Let's have a BP that references asset A
2. Delete asset A, in the popup set asset B to replace existing references

End result: BP is modified (is now referencing asset B), but it's not checked out.

I'm using UE 5.1.1 with plugin version 3.15

**Comment by mastercoms** [comment]
This seems more like an Unreal Engine bug, but I'll make sure about that and see what I can do

**Comment by adamfulop** [comment]
Yeah I see, I was guessing the same but I thought it might worth a report.

**Comment by mastercoms** [comment]
Closing as this can't be fixed in this plugin. I will still see if I can make a PR to the engine eventually.
